api/src/Command/MigrationsMigrateLockCommand.php

```diff
@@ -25,13 +25,6 @@ class MigrationsMigrateLockCommand extends MigrationsMigrateDoctrineCommand
     protected function configure(): void
     {
         parent::configure();
-
-        $this
-            ->setName('doctrine:migrations:migrate-lock')
-            ->setDescription('Same as doctrine:migrations:migrate, but locking the database.')
-            ->setHelp('')
-            ->addOption('release-lock', null, InputOption::VALUE_NONE, 'Release lock and exit.')
-        ;
```

Comment:
Was there an incompatibility reason this was removed? I'm kind of hesitant to stop locking the DB when we carry out migrations...

Reply:
Yeah I removed this to try get the app running as it said the command was invalid. I've added it back now as the `MigrationsMigrateDoctrineCommand` no longer exists but I've changed it to just `Command`